fix readelf output encoding handling of errors

carlosrodfern opened the pull request:
Fixes: #1147

danigm reviewed (approved):
Looks good. Maybe we should add the `errors='replace'` to all the `readelf` calls, we've 4 more calls like this in the `readelfparser.py` file.

carlosrodfern commented:
> Looks good. Maybe we should add the `errors='replace'` to all the `readelf` calls, we've 4 more calls like this in the `readelfparser.py` file.

Thank you @danigm for the review. I added `errors='replace'` to the rest as well and rebased the PR.
